src/extension/background-script/actions/setup/validateAccount.ts

```diff
@@ -12,7 +22,7 @@ const validateAccount = async (message, sender) => {
     return { data: { valid: true, info: info } };
   } catch (e) {
     console.error(e);
-    return { data: { valid: false, error: e.message } };
+    return { data: { valid: false, error: (e as Error).message } };
```

Comment:
In most other places [we do this](https://github.com/getAlby/lightning-browser-extension/blob/feat/add-alias-to-account-name/src/extension/background-script/actions/lnurl/withdraw.ts#L16). Not sure what's better or if we need to be consistent here.

Reply:
Thanks,

I wonder what we should do if e is not an Error? maybe we should have a utility function to handle all the different cases. Otherwise I need to update the error type to be `string | unknown` which I'm not sure we want to deal with downstream

Reply:
`e instanceof Error ? e.message : JSON.stringify(e)` could be another option?